LibraBFT/Impl/Consensus/Liveness/RoundState.agda

```diff
@@ -41,13 +43,28 @@ processLocalTimeoutM now obmEpoch round = do
 
 ------------------------------------------------------------------------------
 
+maybeAdvanceRound : Round → SyncInfo → Maybe (Round × NewRoundReason)
+
 processCertificatesM : Instant → SyncInfo → LBFT (Maybe NewRoundEvent)
-processCertificatesM now syncInfo = do
+processCertificatesM now syncInfo =
+  logEE ("RoundState" ∷ "processCertificatesM" {-∷ lsSI syncInfo-} ∷ []) $ do
   rshcr <- use (lRoundState ∙ rsHighestCommittedRound)
-  if-RWST (syncInfo ^∙ siHighestCommitRound <? rshcr) -- TODO : define and use 'when'
-    then pure unit -- IMPL-TODO ((lRoundState ∙ rsHighestCommittedRound) :=  (syncInfo ^∙ siHighestCommitRound))
-    else pure unit
-  pure nothing
+  when (syncInfo ^∙ siHighestCommitRound <? rshcr) $ do
```

Comment:
@haroldcarr : This should be `>?`.

Reply:
Great catch.   I think I know how this happened.